android/filament-android/src/main/cpp/SurfaceOrientation.cpp

```diff
@@ -20,6 +20,8 @@
 
 #include "common/NioUtils.h"
 
+#include <algorithm>
```

Comment:
Note: we should probably ban `std::min` in favor of `filament::math::min`

Reply:
SGTM

Reply:
I agree. At least in headers.